e2e/login.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {expect, Page, test, TestInfo} from "@playwright/test";
+import {LoginPage} from "../pages/LoginPage";
+import {defaultPassword} from "../helpers/common";
+import {HomePage} from "../pages/HomePage";
+
+interface UsersCredential {
+    email: string;
+    password: string;
+    positive?: boolean;
+}
+
+const testMethod = async (page: Page, testInfo: TestInfo, userCredential: UsersCredential) => {
+    const homepage = new HomePage(page);
+    await homepage.goto();
+    await  homepage.clickOnSignInHeaderButton();
+    const loginPage = new LoginPage(page);
+    await loginPage.fillLoginForm(userCredential.email, userCredential.password);
+    await loginPage.clickOnSubmitButton();
+    if (userCredential.positive) {
+        await loginPage.checkSuccessLogin();
+    } else {
+        await loginPage.takeScreenshotForErrorMessage(`${testInfo.title}-errorMessage.png`);
+        await loginPage.checkNotSuccessLogin();
+    }
+
+
+};
+test.describe('User can work with login', async () => {
+    test('teacher login', async ({page}, testInfo) => {
+        const testUserCredential: UsersCredential = {
+            email: 'vse23688@omeie.com',
+            password: defaultPassword,
+            positive: true,
+        };
+        await testMethod(page, testInfo, testUserCredential);
+    });
+
+
+    test('student login', async ({page}, testInfo) => {
+        const testUserCredential: UsersCredential = {
+            email: 'tafida6678@eimatro.com',
+            password: defaultPassword,
+            positive: true,
+        };
+    });
+
+    test('login user with empty email and password inputs', async ({page}, testInfo) => {
+        const testStudentCredential: UsersCredential = {
+            email: ' ',
+            password: ' ',
+            positive: false,
```

Comment:
Why?
Your test will be work only if block 
if (userCredential.positive) {
        await loginPage.checkSuccessLogin();

Since "if block" work if send any status positive(true or false)

Reply:
Just need to remove positive: false in all tests